app/main.py

```diff
@@ -1,3 +1,16 @@
-def cache(func):
-    # Write your code here
-    pass
+from typing import Callable, Any
+
+
+def cache(func: Callable) -> Any:
```

Comment:
This function returns Callable. I think it's better to specify it: def cache(func: Callable) -> Callable